www/Themes/Generic-theme/style.css

```diff
@@ -2954,3 +2956,14 @@ ul.module_list {
     background-color: var(--gorgone-config-file-background-color);
 }
 
+/* utilities */
+.ms-1 {
+    margin-left: 8px;
+}
+.me-1 {
```

Comment:
Why not `ml` for margin-left and `mr` for margin-right ?

Reply:
s as "start", e as "end". Inspired by Bootstrap 5.

Reply:
I replaced the classes name with yours:
- replace `ms-1` with `ml-1`
- replace `me-1` with `mr-1`